pydm/widgets/base.py

```diff
@@ -849,6 +875,54 @@ def lowerCtrlLimitChanged(self, new_limit):
         """
         self.ctrl_limit_changed("LOWER", new_limit)
 
+    @Slot(float)
```

Comment:
Couldn't it be `float` or `int` here and elsewhere?

Reply:
It looks like even for something like `longin`, the HIHI etc. fields are floats. Bizarre. https://epics.anl.gov/EpicsDocumentation/AppDevManuals/RecordRef/Recordref-21.html

Reply:
That's definitely not the case for the control layer, though. See, for example:
The control type for ints comes back as shorts: https://github.com/klauer/caproto/blob/57e0c564e1463e8a19d7ab6da5d1b9735d04f48f/caproto/_dbr.py#L633-L650
The float version and such are below that.

Reply:
Errors, in the EPICS manual???

Reply:
Nice catch, addressed here: 403e52a3340971fe078fdd1993ce04f41304b4d2